webcamd.c

```diff
@@ -480,6 +481,7 @@ usage(void)
 {
 	fprintf(stderr,
 	    "usage: webcamd -d [ugen]<unit>.<addr> -i 0 -v -1 -B\n"
+	    "	-c Create v4l2loopback device\n"
```

Comment:
```
-c <devname> Create device by given devicename
-c loopback Create v4l2loopback device

```
??

Reply:
Sorry, not sure I understand here. The v4l2loopback code doesn't seem to let you specify the device name, only how many (capped at 8), if that's what you're suggesting?

Reply:
There might be other kinds of loopback devices webcamd might support in the future, not only for video.
For example "evdevloopback" or "dvrloopback". So I suggest that we pass a for now dummy argument, like v4l2loopback for now.

Please also update webcamd.8 with a description of this new option.

Reply:
Makes sense. Is d82cc0f what you had in mind?

Reply:
Perfect. And then fix the pidfile name when v4l2loopback is selected. Maybe just use webcamd.v4l2loopback.pid ??

Reply:
You may also need to check the webcamd rc.d file what it expects too. It is under files/webcamd.in in the port.

Reply:
pidfile name fixed in 612b191. Looks like the script doesn't look at the pidfiles at the moment, just uses pgrep, so no issue there. And if the script changes in the future, it can check all `/var/run/webcamd.*.pid` files.

Reply:
Cool. I've added one more comment. Are there more things we need to consider?

I'd like you to add a comment in the manual page about the option which selects the number of v4l2loopback devices. Might be handy to have it there. What do you think?

Reply:
Yeah, I was thinking that too, done. Not sure what else to consider right now. FWIW, I've tested with obs-v4l2sink writing to the v4l2loopback device and Firefox reading it, seems fine so far. Maybe let others try it out and see what they run into?

Reply:
Then after you squash those changes I'll add another webcamd tag.
Do you want to update the webcamd port for me?

Reply:
Sure, updating the port is no problem.